BepInEx.PatcherProvider/BepInExPatcherProvider.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using BepInEx.Logging;
+using BepInEx.Preloader.Core.Patching;
+
+namespace BepInEx.PatcherProvider;
+
+[PatcherProviderPluginInfo("io.bepinex.bepinexpatcherprovider", "BepInExPatcherProvider", "1.0")]
```

Comment:
I know `bepinex.io` is used as id in other places, but we don't own that domain, so I don't think we should use it. Just `BepInExPatcherProvider` would be fine probably.